packages/shared/src/hooks/useCreateSquadModal.ts

```diff
@@ -26,14 +23,14 @@ export const useCreateSquadModal = ({
   isFlagsFetched,
 }: UseCreateSquadModalProps): UseCreateSquadModal => {
   const router = useRouter();
-  const { openModal } = useLazyModal();
   const [hasTriedOnboarding, setHasTriedOnboarding, isLoaded] =
     usePersistentContext<boolean>(SQUAD_ONBOARDING, hasSquads);
 
   // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
```

Comment:
Is this still needed here?

Reply:
Refactored the whole hook in 91333732c13665fe09ef49056ce8d64255124d38